packages/server/__tests__/api/utils.js

```diff
@@ -2,10 +2,15 @@ const fetch = require('node-fetch');
 require('dotenv').config();
 const { knex } = require('../../src/db');
 
+function getTestDomain() {
+    const { PORT = 3000 } = process.env;
```

Comment:
Is this equivalent to `if process.env.PORT) { PORT = process.env.PORT } else { PORT = 3000 )` ?

Reply:
Yep. This syntax is called [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) with a default value.